src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2310,8 +2311,8 @@ def webapp_up(cmd, name, location=None, sku=None, dryrun=False, logs=False):  #
     location = set_location(cmd, sku, location)
     loc_name = location.replace(" ", "").lower()
     is_linux = True if os_val == 'Linux' else False
-    asp = "{}_asp_{}_{}".format(user, os_val, loc_name)
-    rg_name = "{}_asp_{}_{}".format(user, os_val, loc_name)
+    asp = "{}_asp_{}_{}_0".format(user, os_val, loc_name)
```

Comment:
any reason you want to start with `0`? Usually the name sequences would be `name`, `name_2`, etc

Reply:
0 seems like a good # to start 1 or 0 is something was thinking about. if someone already has 2 then we will go 3 as per this logic

Reply:
this is the schema i was thinking - for ASP the RG can have multiple ASPs with different SKUs
![image](https://user-images.githubusercontent.com/13839372/56330050-46c82f80-613b-11e9-8bb9-f74e8e1ce03c.png)

Reply:
Did not want to use SKU in the name because customers usually scale up or down to different SKU configurations